SUPPORT.md

```diff
@@ -0,0 +1,34 @@
+<!--
+SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# Getting Help
+
+There are a few ways to connect with the `power-grid-model` project:
+
+* Submit an issue
+* Send an email to the Technical Steering Committee: <dynamic.grid.calculation@alliander.com>.
+
+## How to ask for help, suggest a feature, or give recommendations.
+
```

Comment:
This section would be different but similar to one in power-grid-model

https://github.com/alliander-opensource/power-grid-model/blob/main/docs/release_and_support/SUPPORT.md#how-to-ask-for-help-suggest-a-feature-or-give-recommendations

Reply:
Good feedback. I will update this.